CMakeLists.txt

```diff
@@ -189,14 +199,6 @@ target_include_directories(evhtp PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
 )
 
-if(BUILD_SHARED_LIBS)
-    set_target_properties(evhtp PROPERTIES
-        VERSION "${PROJECT_VERSION}"
-        SOVERSION 0
-        OUTPUT_NAME "evhtp"
-        C_VISIBILITY_PRESET hidden)
-endif()
```

Comment:
This is just about the same thing as the Hunter devs did; I don't see a really good reason why to remove this version.